packages/kit/test/apps/basics/test/cross-platform/client.test.js

```diff
@@ -221,15 +221,24 @@ test.describe('Navigation lifecycle functions', () => {
 	});
 
 	test('beforeNavigate is triggered after clicking a download link', async ({ page, baseURL }) => {
-		await page.goto('/navigation-lifecycle/before-navigate/prevent-navigation');
+		const pathname = '/navigation-lifecycle/before-navigate/prevent-navigation';
 
-		await page.click('a[download]');
-		expect(await page.innerHTML('pre')).toBe('0 false undefined');
+		await page.goto(pathname);
+
+		const download = page.waitForEvent('download', { timeout: 3000 });
```

Comment:
do we need to override the default timeout here?

Reply:
I can't remember if I added that because it was too short or too long. Let me try removing it.